kevm-pyk/src/kevm_pyk/kproj/evm-semantics/serialization.md

```diff
@@ -55,93 +46,12 @@ Address/Hash Helpers
-                                                  , HF
-                                                  ]
-                                                )
-                                    )
-                         )
-
-    rule #blockHeaderHash(HP:Bytes, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF)
-         => #parseHexWord( Keccak256( #rlpEncode( [ HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF ] ) ) )
-
-    rule #blockHeaderHash(HP:Int, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF)
-         => #parseHexWord( Keccak256( #rlpEncode( [ #wordBytes(HP), #wordBytes(HO), #addrBytes(HC)
-                                                  , #wordBytes(HR), #wordBytes(HT), #wordBytes(HE)
-                                                  , HB, HD, HI, HL, HG, HS, HX
-                                                  , #wordBytes(HM), #padToWidth(8, #asByteStack(HN))
-                                                  , HF , #wordBytes(WF)
-                                                  ]
-                                                )
-                                    )
-                         )
-
-    rule #blockHeaderHash(HP:Bytes, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF, UB, EB, BR)
-         => #parseHexWord( Keccak256( #rlpEncode( [ HP, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF, UB, EB, BR] ) ) )
-
-    rule #blockHeaderHash(HP:Int, HO, HC, HR, HT, HE, HB, HD, HI, HL, HG, HS, HX, HM, HN, HF, WF, UB, EB, BR)
-         => #parseHexWord( Keccak256( #rlpEncode( [ #wordBytes(HP), #wordBytes(HO), #addrBytes(HC)
-                                                  , #wordBytes(HR), #wordBytes(HT), #wordBytes(HE)
-                                                  , HB, HD, HI, HL, HG, HS, HX
-                                                  , #wordBytes(HM), #padToWidth(8, #asByteStack(HN))
-                                                  , HF , #wordBytes(WF) , UB , EB , #wordBytes(BR)
-                                                  ]
-                                                )
-                                    )
-                         )
-```
 
 - `#hashTxData` returns the Keccak-256 message hash `HT` to be signed.
 The encoding schemes are applied in `#rlpEcondeTxData`.
 
 ```k
     syntax Bytes ::= #hashTxData ( TxData ) [symbol(hashTxData), function]
- // ----------------------------------------------------------------------
-    rule #hashTxData( TXDATA ) => Keccak256raw(                #rlpEncodeTxData(TXDATA) ) requires isLegacyTx    (TXDATA)
-    rule #hashTxData( TXDATA ) => Keccak256raw( b"\x01" +Bytes #rlpEncodeTxData(TXDATA) ) requires isAccessListTx(TXDATA)
-    rule #hashTxData( TXDATA ) => Keccak256raw( b"\x02" +Bytes #rlpEncodeTxData(TXDATA) ) requires isDynamicFeeTx(TXDATA)
-    rule #hashTxData( TXDATA ) => Keccak256raw( b"\x03" +Bytes #rlpEncodeTxData(TXDATA) ) requires isBlobTx      (TXDATA)
```

Comment:
We may need these rules, as you kept the definition on the file.

Reply:
The PR is updated: these rules are restored.